eng/Publishing.props

```diff
@@ -1,5 +1,37 @@
 <Project>
-   <PropertyGroup>
-      <ProducesDotNetReleaseShippingAssets>true</ProducesDotNetReleaseShippingAssets>
-   </PropertyGroup>
-</Project>
\ No newline at end of file
+
+  <PropertyGroup>
+    <ProducesDotNetReleaseShippingAssets>true</ProducesDotNetReleaseShippingAssets>
+    <PublishDependsOnTargets>$(PublishDependsOnTargets);_PublishInstallers</PublishDependsOnTargets>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_InstallersToPublish Remove="@(_InstallersToPublish)" />
```

Comment:
WindowsDesktop may be special here. I would find out how they are publishing their assets right now.

Reply:
Ideally this change would replace https://github.com/dotnet/windowsdesktop/blob/main/src/publish/prepare-artifacts.proj but that might need a bit more work.

Otherwise we would still publish differently. This applies to windowsdesktop and runtime. Both use prepare-artifacts.proj.